packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -241,17 +242,8 @@ module.exports = {
   extraWatchFiles: [
     '.vuepress/nav/*',
   ],
+  
   additionalPages: [
     ...guidesInfo.additionalPagesForGuides(),
   ],
-  extendPageData(page) {
```

Comment:
Have you tested this in both `yarn dev` and a `yarn build` with a static server? (to test both the SSR and the SPA code?)

A common problem case was loading a guide and following links in it to other guides versus loading those links directly.  (going to a link directly relies on SSR, while following a link uses the SPA code).  

Often one or the other would break with guides.

Reply:
ah I see. I haven't tried with the static server (yarn build). I'll try it out and report back here.

Reply:
Tested the SSR and SPA flows. Both are working!